omaha-client/examples/hello-world/main.rs

```diff
@@ -32,13 +33,21 @@ mod policy;
 mod storage;
 mod timer;
 
-/// Service endpoint of the omaha server to connect to.
-/// This service must be ready to respond for this example program to work.
-const SERVICE_URL: &str = "http://[::1]:35373";
 /// App ID of the application for which an update is checked / requested.
 /// The omaha service must know this app ID for this example program to work.
 const APP_ID: &str = "appid_01";
 
+#[derive(FromArgs)]
+/// Arguments for the omaha-client hello world example
+struct Args {
+    #[argh(
+        option,
+        description = "URL of the omaha service to connect to. This service must accept connections.",
+        default = "\"http://[::1]:35373\".to_string()"
```

Comment:
Since we're now starting on a random port, how about getting rid of the default and making the user specify the port of the mock-omaha-server?

Reply:
Done. Also added a short flag.